src/app/core/core.component.html

```diff
@@ -1,11 +1,18 @@
 <div defaultOverlayTarget></div>
-<cut-phase-bar phaseLabel="BETA" phaseLink="'javascript:void(0)'"></cut-phase-bar>
+
+<!-- BETA -->
+<cut-phase-bar *ngIf="!isSolicitor()" phaseLabel="BETA" phaseLink="'javascript:void(0)'"></cut-phase-bar>
+
+<!-- Header : Start -->
 <cut-header-bar [title]="selectedJurisdictionName"
-  username="{{profile?.user.idam.forename | titlecase}} {{profile?.user.idam.surname | titlecase}}"
-  (signOutRequest)='signOut()'></cut-header-bar>
-<cut-nav-bar class="global-navigation">
-  <div class="nav-left" leftNavLinks>
-    <ul id="menu-links-left">
+                [isSolicitor]="isSolicitor()"
+                username="{{profile?.user.idam.forename | titlecase}} {{profile?.user.idam.surname | titlecase}}"
+                (signOutRequest)='signOut()'>
+
+  <!-- Header menu : Start -->
+  <!-- "true" condition is placeholder whether to display header menu or not -->
+  <div *ngIf="true" headerNavigation>
```

Comment:
what if we introduced a method in the component displayHeader() that for the moment just returns true? It'll be more self explanatory and we could also remove the explanatory comment